### ISSUE TITLE
Better Distinction Between LiDAR Results

### ISSUE THREAD

**Issue by stdavis** [agrc/raster#168]
I wonder if there is a way to make it easier for the user to know which one of these ".5 Meter" results to choose without opening them all up first?
![image](https://github.com/agrc/raster/assets/1326248/b3cd595a-8b1c-4757-89a9-60b6818d6256)

This may be related to the question of data first vs. project first results.

**Comment by rkelson** [comment]
I did not know clicking to get more information was so intrusive. Shoot we just got a google gold star for number of clicks when one would think less clicks navigating a website is better but I live in a different world.

Here is where there is some disconnect with actual usage intentions and development testing trying to shot holes in a proven success story to push a new narrative. The app was never intended for people to query an area that large. Even if there was one project that covered that whole area you would get this error when trying to download the lidar:
![image](https://github.com/agrc/raster/assets/6200941/073b582a-6f8e-4ae1-935c-9439cec9f452)

In my world it's more valuable to know everything that is available for an area over time rather than someone behind the curtain deciding what I will be provided with. While that might work for some or even most, this app is intended to provide a collection of available information and not so much 'here are 3 clicks to make pretty pictures of Timp'. I guess my intentions  are more to provide information to solve peoples problem vs. making pretty pictures. For instance a couple weeks ago a guy was excited there were 3 different collections for his AOL to compare with the river flows at the exact time the lidar was collected.

And then there is the scale argument, and wouldn't the 10 meter actually work better for such a large area.

And then there is the rule, "no matter what size and shape of spoon you try and build for people, someone is going to want something different". I'm not sure exactly what Jake is intending and there is the rub, I've been intentionally left out of the discussions so he can push his idea.

**Comment by stdavis** [comment]
OK. I can see that my test may not be a great reflection of a real-world use case. I just wondered if there was a way to give the user more info at the result level to help them get to what they are interested in more quickly. Not a huge deal. I opened an issue to make sure that we considered this during the rewrite.

The conversation about Jake's idea of a statewide LiDAR layer probably deserves its own issue. @jacobdadams, do you want to create one in this repo?

It's the same case for the data first vs project first presentation question. I wasn't sure that could put that one into words yet so I didn't create an issue for it.

> I've been intentionally left out of the discussions

I feel bad that you feel this way. I don't know of any meetings or meaningful discussions about this app that you have been left out of. Yesterday was the first that I heard of Jake's idea. I still certainly see you as the subject matter expert for this app.

**Comment by rkelson** [comment]
I was simply told there were discussions going on 'behind my back' that shouldn't be happening is all.

The lidar is tough to add more useful information to the tabs to differentiate the products because of how it had to be collected.

Sounds like Jakes quilter is a one off per user but a whole new dataset of large mosaiced tiles could be created for those who don't care about from whence it came. Maybe he plans on both but he is conveniently not talking to me about either and simply states 'we are doing this' which is out of line. Quilter sounds like technical debt that might not be worth it. My original intention with the lidar was for the smaller scale technical folks solving Utah's problems and not the artsy folks.

**Comment by nathankota** [comment]
> I was simply told there were discussions going on 'behind my back' that shouldn't be happening is all.

The idea has been swirling for a while and Jake has expressed the idea to me and we have talked briefly about it in the past, but it is only now starting to surface, so please rest assured that you haven't missed out on or not been included in any discussions. I apologize if I have misled you into thinking there was talk behind your back.

> Maybe he plans on both but he is conveniently not talking to me about either and simply states 'we are doing this' which is out of line. 

 I recommend that we stay focused on continuing to hone in on the major goals of the project (some of which are below), consider this as its own goal and add a new issue for it as Scott suggested, and discuss its merits and make a decision as a team.

> Broad project goals:
> - Move to the  cloud
> - Make publishing data easier
> - Make downloading easier 
> - Make the products that are available more understandable for less technical people
> - Downloading the data may be a little confusing
> - Make managing the data easier 
> - Update the API
> - ...

**Comment by jacobdadams** [comment]
Rick, I apologize for making you feel left out of the discussions about a larger, "cohesive" lidar dataset. That was definitely not my intention. You're definitely the lidar expert and I did not and do not intend to minimize that or try to keep you in the dark.

Like Nathan said, this new dataset is just something that's been bouncing around in the back of my head ever since I started playing with AGRC/UGRC's lidar. In my yearly UPM meetings with Nathan I've set one of my "stretch" goals to be investigating whether it's possible and would make sense. Beyond those conversations, it's just one of those things I think about in the shower. 

I brought it up in the discussion yesterday and pushed hard for it because a) we're nearing total state-wide coverage and b) it's something I've wanted to do for several years but it keeps getting pushed aside by other projects, so I'm hoping that this work can finally carve out some time to make it happen.

I whole heartedly agree with you that we shouldn't hide the individual projects. There are a lot of use cases and benefits to having them available. This new proposed dataset would very much be a derivative and modified product and thus not the most accurate product. 

Quilter was my first project here at UGRC to help people download multiple tiles of raster data. It never made it out of development and is, frankly, a hot mess. My next development step for it (if we wanted to continue it) would be "delete and start over."

I do believe there's value in a cohesive statewide dataset as another spoon for people, so I'll open another issue related to that so we can have a proper discussion about it now that we've got resources/time to do so.

**Comment by rkelson** [comment]
While I still have not come around to a 'quilter button' I am behind a new statewide UGRC 1 Meter dataset and that is why I started messing with the national grid to come up with some tiling ideas. I was really surprise with my first test to merge ~1200 tiles in that it took less then 10 minutes to output a ~9gb file from Global Mapper. It is possible the data could be more compressed but it was just a first test run where I expected it to blow up. We should talk more but my idea to create 106-50Km/sq tiles which seems reasonable and I think it cold solve a lot of folk's pain points.

Here is an image of what I am talking about. I created topology on the lidar extents and the few large pink areas denote project overlap where we would choose which dataset would be used and it could also become sort of a metadata to track the lineage of the new 1 meter dataset:
![image](https://github.com/agrc/raster/assets/6200941/b217533c-7c69-4568-a502-f36061b9175c)

**Comment by stdavis** [comment]
Yes, we can definitely concat fields on the fly to label results. We should definitely take a fresh look at this when we start the design for v3.

**Comment by rkelson** [comment]
We could add the year after the resolution

**Comment by rkelson** [comment]
Looking great to me!
